plugin/pkg/auth/authorizer/node/graph_populator.go

```diff
@@ -48,15 +46,6 @@ func AddGraphEventHandlers(
 
 	var hasSynced []cache.InformerSynced
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) {
-		nodes.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-			AddFunc:    g.addNode,
-			UpdateFunc: g.updateNode,
-			DeleteFunc: g.deleteNode,
```

Comment:
I tried and found more dependencies. I will do a follow up PR, want to keep this to minimum. If it will be better I can undo all changes in this file.